developer_manual/app/publishing.rst

```diff
@@ -33,6 +33,7 @@ Requirements:
 * security audited and design reviewed
 * app is at least 6 months old and has seen regular releases
 * follows app guidelines
+* supports the same platforms, browsers, databases, PHP versions and so on as ownCloud Core
```

Comment:
»core« is lowercase, right? Also we need to link to the page where these requirements are listed. If that doesn’t exist, we need to create it. :D